breathecode/registry/views.py

```diff
@@ -614,6 +614,8 @@ def get(self, request, asset_slug=None):
 
         if 'big' in self.request.GET:
             serializer = AssetMidSerializer(items, many=True)
+        elif 'with_techs' in self.request.GET:
+            serializer = AssetAndTechnologySerializer(items, many=True)
```

Comment:
Test it

You must use the PyTest style:
- The current fixtures: you can read this to understand how to write fixtures https://github.com/breatheco-de/apiv2/blob/development/conftest.py
- A test that use this style: see in the start it use `pytest.mark.usefixtures('db')` to allow to all tests use the database and then `setup` is being autouse in all the test, `monkeypatch` is applying the mock in all tests https://github.com/breatheco-de/apiv2/blob/development/breathecode/payments/tests/admin/tests_add_cohort_set_to_the_subscriptions.py so you can apply with `pytest.mark.usefixtures` any fixture globally in this test file
- `self.bc` is a fixture called `bc` now ( `def test_no_cohort_sets(bc: Breathecode):` )
- The test style are functional, so, `self.assertEqual` does not exist, use `assert bc.database.list_of('payments.Subscription') == []` style instead
- If you need to see more details you can use `-v`, `-vv`, `-vvv`

I recommend you to use this if you don't wanna write 30 tests:
- Lookup extension: https://github.com/breatheco-de/apiv2/blob/8705f9e2d9f4cd6eeb3dce05163c0f8cd2abe73d/breathecode/events/views.py#L237
- Test: https://github.com/breatheco-de/apiv2/blob/development/breathecode/events/tests/urls/tests_me_event_liveclass.py